druid/src/theme.rs

```diff
@@ -50,14 +50,14 @@ pub const BORDERED_WIDGET_HEIGHT: Key<f64> = Key::new("bordered_widget_height");
 
 pub const TEXTBOX_BORDER_RADIUS: Key<f64> = Key::new("textbox_radius");
 
-pub const SCROLL_BAR_COLOR: Key<Color> = Key::new("scroll_bar_color");
-pub const SCROLL_BAR_BORDER_COLOR: Key<Color> = Key::new("scroll_bar_border_color");
-pub const SCROLL_BAR_MAX_OPACITY: Key<f64> = Key::new("scroll_bar_max_opacity");
-pub const SCROLL_BAR_FADE_DELAY: Key<u64> = Key::new("scroll_bar_fade_time");
-pub const SCROLL_BAR_WIDTH: Key<f64> = Key::new("scroll_bar_width");
-pub const SCROLL_BAR_PAD: Key<f64> = Key::new("scroll_bar_pad");
-pub const SCROLL_BAR_RADIUS: Key<f64> = Key::new("scroll_bar_radius");
-pub const SCROLL_BAR_EDGE_WIDTH: Key<f64> = Key::new("scroll_bar_edge_width");
+pub const SCROLLBAR_COLOR: Key<Color> = Key::new("scrollbar_color");
+pub const SCROLLBAR_BORDER_COLOR: Key<Color> = Key::new("scrollbar_border_color");
+pub const SCROLLBAR_MAX_OPACITY: Key<f64> = Key::new("scrollbar_max_opacity");
+pub const SCROLLBAR_FADE_DELAY: Key<u64> = Key::new("scrollbar_fade_time");
+pub const SCROLLBAR_WIDTH: Key<f64> = Key::new("scrollbar_width");
```

Comment:
one note, that isn't especially related to this PR; these keys should all be namespaced or otherwise made more unique. Elsewhere in the project i use `druid.built-in.your-key-here` for this. 🤷‍♂

Reply:
I just did a find-replace and read through the diff lol so I didn't make any structural changes to anything like this as that would have been out of scope even had I know this

Reply:
yep, just saw  this and wanted to share some project domain knowledge that isn't written down anywhere 😬